src/Dropzone/README.md

```diff
@@ -85,7 +85,7 @@ Symfony UX Dropzone allows you to extend its default behavior using a custom Sti
 ```js
 // mydropzone_controller.js
 
-import { Controller } from '@hotwired/stimulus';
+import { Controller } from 'stimulus';
```

Comment:
@hotwired/stimulus is good package name, see https://www.npmjs.com/package/@hotwired/stimulus

Reply:
thanks for your message.
I have follow the instructions of README on symfony 5.3 and he said '@hotwired/stimulus' error (i have run `yarn install --force` but if i change by stimulus, i have no problem.

Reply:
I believe tutorials were not updated, what happens if you run `yarn add @hotwired/stimulus`?

Reply:
it's ok but it's a supplement library ;-)

Reply:
AFAIK it's the same library than `stimulus`, it has been renamed.

Reply:
ok thanks ;-) but the error exist.

Reply:
Not really, since #141, Symfony UX packages have been updated to Stimulus 3 and has [peer dependency to @hotwired/stimulus](https://github.com/symfony/stimulus-bridge/blob/main/package.json#L16). 

Also, please take a look to https://github.com/symfony/stimulus-bridge/blob/main/CHANGELOG.md#300, it explains what changed with the new Symfony's Stimulus bridge.